src/index.css

```diff
@@ -0,0 +1,420 @@
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  background: rgba(0, 0, 0, 0.05);
+  padding: 2rem;
+}
+
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+/* {* NAVBAR STYLES *} */
+
+.navbar {
+  display: flex;
+  height: 6rem;
+  width: 87.5rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: var(--primary-white-2);
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.5);
+}
+
+.navlink-books {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--primary-black);
+  text-decoration: none;
+  margin-left: 3rem;
+}
+
+.dev-status {
+  width: auto;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 1.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--primary-black);
+  text-decoration: none;
+  margin-left: 3rem;
+}
+
+.navlink-categories,
+.back-home {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  opacity: 0.5;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--primary-black);
+  text-decoration: none;
+}
+
+.back-home {
+  width: auto;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  opacity: 0.8;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--primary-blue);
+  text-decoration: none;
+  white-space: nowrap;
+}
+
+.nav {
+  display: grid;
+  grid-template-columns: auto auto 1fr;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.account {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 1rem;
+  color: var(--primary-blue);
+}
+
+.logo {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-size: 1.5rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--primary-blue);
+  white-space: nowrap;
+}
+
+/* FORM */
+.form {
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.title {
+  width: 41.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 35.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: var(--primary-white-2);
+}
+
+.author {
+  width: 17.813rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: var(--primary-white-2);
+}
+
+.form-title {
+  width: 10.875rem;
```

Comment:
- [ ] In addition, I would also suggest that you style the Book title, author, and comment sections to look exactly like the model design as provided on Zeplin.

- [ ] I would also suggest you use the correct font family, size, and weight on the logo. You can use the font finder Chrome extension to help you identify the font characteristics, that way you can choose the correct font for this task.

- [ ] Kindly make sure the form aligns accurately with the card components. This will greatly improve the outlook of your design

> Zeplin design
![proper design](https://user-images.githubusercontent.com/71140133/230510331-aba1e11f-a27f-4308-8a4a-7b616dc05059.PNG)

> Your design
![wrong design](https://user-images.githubusercontent.com/71140133/230510388-3aceb27d-0861-4c26-a6d9-2da6138b4dbf.PNG)

- Kindly adjust the width of your form inputs to look exactly like the model design provided.

> Zeplin design
![reaL](https://user-images.githubusercontent.com/71140133/230510978-4fedc300-b453-4a28-a1b4-2a1c9d62a1bf.PNG)

Reply:
@Yerimah Thank you for all the references you suggested,they were very insightful and helped me greatly in implementing the changes as required👌